docs: labs reorg (#745)
redpanda-data/documentation: docs/labs/index.mdx

---
title: Labs
---

<head>
    <meta name="title" content="Labs | Redpanda Docs"/>
    <meta name="description" content="Try new features"/>
</head>

> Redpanda labs are product experiments. We'd love for you to test them and give us your feedback. 

Reviewer: ```suggestion
Redpanda labs are product experiments. We encourage and welcome you to test them and share your feedback. 
```

Reviewer: Please make this update for sure.

- [Install Labs](./install-preview)
- [Data Transforms](./data-transform)

> Send us your feedback on our [Slack Community](https://join.slack.com/t/redpandacommunity/shared_invite/zt-ng2ze1uv-l5VMWSGQHB9gp47~kNnYGA/) or [GitHub Discussions](https://github.com/vectorizedio/redpanda/discussions).  

Reviewer: ```suggestion
Send your feedback on our [Slack Community](https://join.slack.com/t/redpandacommunity/shared_invite/zt-ng2ze1uv-l5VMWSGQHB9gp47~kNnYGA/) or [GitHub Discussions](https://github.com/vectorizedio/redpanda/discussions).  
```

:::caution

> Lab features may be tied to specific version releases of the product and are not supported in production deployments.

Reviewer: ```suggestion
Lab features may be tied to specific product release versions and are not supported in production deployments.
```
